packages/drivers/odsp-driver/src/compactSnapshotParser.ts

```diff
@@ -54,9 +60,9 @@ function readAndValidateHeaderSection(node: NodeCore): ISnapshotHeader {
  */
 function readDictionarySection(node: NodeCore) {
     const dictionary = new Array<string>();
-    for (const name of node) {
-        assert(name instanceof BlobCore, 0x211 /* "Mapping should be of type BlobCore" */);
-        dictionary.push(name.toString());
+    for (const childNode of node) {
+        assertBlobCoreInstance(childNode, "Mapping should be of type BlobCore");
```

Comment:
We can decide it later, but we should consider if we want each instance of assertBlobCoreInstance to have unique assert tag. I think the answer is yes, as strings add up a lot to bundle.
The way we solve it in other places is a project (package) to have a file that lists all the names of APIs that needs to be tagged, and tagger understands it and can do it not just for "assert" but for all other names.
Can you please open a bug to follow up here?

@anthony-murphy - FYI.

Reply:
Created an issue  here: https://github.com/microsoft/FluidFramework/issues/7185